### ISSUE TITLE
problem importing legacy vtk file with 2D vectors

### ISSUE THREAD

**Issue by schruste** [nschloe/meshio#935]
Hi, 

I have a problem with a series of legacy vtk-files that I would have liked to convert (to vtu). However, reading in the vtk file fails already. A minimal VTK file that already fails is as simple as this:

```
# vtk DataFile Version 3.0
vtk output
ASCII
DATASET UNSTRUCTURED_GRID
POINTS 6 float
       0       1	 0.0
       0       0	 0.0
       1       0	 0.0
       0       1	 0.0
       1       0	 0.0
       1       1	 0.0
CELLS 2 8
3	0	1	2	
3	3	4	5	
CELL_TYPES 2
5 
5 
CELL_DATA 2
POINT_DATA 6
SCALARS sol float 2
LOOKUP_TABLE default
-1 1 -1 0 0 -0 0 0 0 0 0 0 
```

Thanks for the convenient tool and your effort. 

Best,
schruste

**Comment by schruste** [comment]
Thanks!

**Comment by nschloe** [comment]
Thanks for the great bug report. Fixed in 4.2.2.
